src/gui/addnewtorrentdialog.ui

```diff
@@ -96,6 +96,16 @@
     </widget>
    </item>
    <item>
+    <widget class="QCheckBox" name="doNotDeleteTorrentCheckBox">
+     <property name="toolTip">
+      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When checked, the .torrent file will not be deleted despite the settings at the &amp;quot;Download&amp;quot; page of the options dialog.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
Better set this Tooltip in the code in order to strip out the html tags and prevent translators from borking it. eg Tooltip = start_tags + tr(tooltip plaintext) + end_tags;